### PR TITLE
Add Learn Content on Run and Debug 

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add Learn content on Run and Debug.
> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+>**Info** The Ballerina debugger is an open-source project and contributors are mostly welcome to get engaged with it via the [ballerina-lang](https://github.com/ballerina-platform/ballerina-lang) GitHub repository. If you encounter any difficulties when using this feature, feel free to create an issue in it.
+
+For more information on the above features, see [Using the Debugging Features](/learn/tooling-guide/visual-studio-code-extension/run-and-debug/using-the-debugging-features/).
+
+## Troubleshooting
+- Stepping over code lines in non-blocking paths (eg: action invocations) will not pause VM on the next line
+    - workaround: manually put a breakpoint to the next line
+- There are some cases where stepping over gives unexpected behavior
+    - E.g., when there are multiple workers and a wait expression waiting for them, even though it steps over to hit and pass the wait line in the source, the workers may not have finished the execution yet.
```
Comment (NipunaRanasinghe) [inline learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:95]: Shall we remove this troubleshooting content, as it's outdated for Swan Lake?

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/using-the-debugging-features.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+The Ballerina VSCode debugger lets you evaluate expressions in the ways below.
+
+### Using the Debug Console
+
+### Using the Watch Window
+
+### Existing Limitations
+
+The below are the Ballerina expression types that are currently supported by the debugger expression evaluator and the existing limitations.
```
Comment (NipunaRanasinghe) [inline learn/tooling-guide/visual-studio-code-extension/run-and-debug/using-the-debugging-features.md:39]: Lets remove this sentence as well

**Review by NipunaRanasinghe — APPROVED** [review]

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md (hunk 1 above)
Comment (praneesha) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:95]: Removed.

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/using-the-debugging-features.md (hunk 2 above)
Comment (praneesha) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/using-the-debugging-features.md:39]: Removed.

**Pull request merged**